app/views/memberships/_membership.html.erb

```diff
@@ -22,7 +22,13 @@
     </time>
     <br>
     <% if membership.invitation.present? %>
-        Invited by: <%= membership.invitation.invitor.name %>
+      Invited by <%= membership.invitation.invitor.name %>
+      on
+      <br />
```

Comment:
I think in 2022 we no longer self-close <br> tags? Or is it the other way around? Not that it matters functionally. Only matters OCD-ally.